lib/applicaster/test/accounts/mock_data.rb

```diff
@@ -13,6 +13,18 @@ def all_accounts_attributes
             }
           end
         end
+
+        def user_attributes
+          @user_attributes ||= begin
```

Comment:
I don't really understand the rational here?
So I cannot reference two different users in the same test?